src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -8,11 +8,13 @@
 import org.usfirst.frc4904.standard.CommandRobotBase;
 import org.usfirst.frc4904.standard.commands.chassis.ChassisIdle;
 import org.usfirst.frc4904.standard.commands.chassis.ChassisMove;
+import edu.wpi.first.wpilibj.command.Command;
 import edu.wpi.first.wpilibj.command.Subsystem;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 public class Robot extends CommandRobotBase {
 	RobotMap map = new RobotMap();
+	Command autoShifter;
```

Comment:
Should this maybe be in `HumanInterfaceConfig` if it is shared?
Also, why not make the type AutoShifter?